components/Checkbox/Checkbox.css

```diff
@@ -50,6 +50,10 @@
   background: var(--color-white);
 }
 
+.input:focus + .label:before {
+  box-shadow: 0 0 3px 1px var(--color-blue);
```

Comment:
The focus colour we use on `FormInput` is the teal. We should keep them consistent — either way.